hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/RpcServer.java

```diff
@@ -496,11 +515,44 @@ void logResponse(Message param, String methodName, String call, String tag,
           }
         }
       }
-      responseInfo.put("multi.gets", numGets);
-      responseInfo.put("multi.mutations", numMutations);
-      responseInfo.put("multi.servicecalls", numServiceCalls);
+      responseInfo.put(MULTI_GETS, numGets);
+      responseInfo.put(MULTI_MUTATIONS, numMutations);
+      responseInfo.put(MULTI_SERVICECALLS, numServiceCalls);
     }
+    final String tag = tooLarge ? "TooLarge" : "TooSlow";
     LOG.warn("(response" + tag + "): " + GSON.toJson(responseInfo));
+    if (!tooLarge && this.onlineSlowLogProvider != null
```

Comment:
Are you nog handling tooLarge cases here?

Reply:
Correct, we are just handling tooSlowResponses as per the Jira

Reply:
Since you're doing all this good work to get the `slowResponse` log support, we should consider following it with support for `largeResponse` messages as well? Seem like most of the hard stuff will be resolved by the first feature. Maybe follow-on JIRA.